### PR TITLE
[OSB-8] fix: Update README and Web examples with npmjs packages

### PR DESCRIPTION
**Pull request by MAG-DanielMurdolo**
connected-spaces-platform Pull Request

**For the reviewer**

* [ ] If required, are the changes covered by appropriate tests?
* [ ] Are any public-facing API changes well documented?
* [ ] Is the code easily readable and extensible and/or follow existing conventions?

### CODE REVIEW

**Review by MAG-SamBirley — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,37 +18,42 @@ _"An interoperable communication library for the spatial internet."_
...
 ## 📦 Install
-NPM packages are available for Web and Unity and can be found in [Packages](https://github.com/orgs/magnopus-opensource/packages?repo_name=connected-spaces-platform).
 
-- Web Installation:  `npm install @magnopus-opensource/connected-spaces-platform.web`
+NPM packages are available for Web and Unity and can be found on npmjs.
 
-- Unity Installation: `npm install @magnopus-opensource/connected-spaces-platform.unity.core`
+- Web: https://www.npmjs.com/package/connected-spaces-platform.web
+
+- Unity: https://www.npmjs.com/package/connected-spaces-platform.unity.core
```
Comment (MAG-SamBirley) [inline README.md:47]: Can we add the following after the URL:

(for more information on Unity pacakge installation, head over [here](https://github.com/magnopus-opensource/connected-spaces-platform/wiki/Using-CSP-For-Unity))

**Review by MAG-SamBirley — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MAG-DanielMurdolo) [reply README.md:47]: image

Thoughts?

Comment (MAG-SamBirley) [reply README.md:47]: Perfect

Comment (MAG-SamBirley) [reply README.md:47]: Although end the sentence with a full stop please.

Comment (MAG-DanielMurdolo) [reply README.md:47]: Done

**Review by MAG-SamBirley — APPROVED** [review]

**Review by MAG-AdamThorn — APPROVED** [review]

**Pull request merged**